src/components/__tests__/PluginContent.test.jsx

```
@@ -0,0 +1,153 @@
+import React from 'react';
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)